core/src/main/protobuf/org/apache/spark/status/protobuf/store_types.proto

```diff
@@ -0,0 +1,60 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+syntax = "proto3";
+package org.apache.spark.status.protobuf;
+
+message JavaDate {
+  // The number of milliseconds since January 1, 1970, 00:00:00 GMT
+  optional int64 date = 1;
+}
+
+enum JobExecutionStatus {
+  RUNNING = 0;
+  SUCCEEDED = 1;
+  FAILED = 2;
+  UNKNOWN = 3;
+}
+
+message JobData {
+  int32 job_id = 1;
```

Comment:
Discuss:

Since we are defining the schema here, do we want to make all id's (rdd, stage, job, etc) `int64` instead of `int32` ?
This will be a pain to change in future for protobuf, if we need to - unlike json.

(Example, see [this PR](https://github.com/apache/spark/pull/38781)).

Reply:
ok, changed to int64